project.clj

```diff
@@ -100,6 +100,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.5"]
+                                   [org.clojure/java.jmx "0.2.0"]
```

Comment:
can you look at the pom.xml in your .m2 and see if it's in there?  if not, maybe google for 15 mins and then punt and make it a regular dep?